packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -76,9 +75,11 @@ class ChromeLauncher {
   ///
   /// `headless` defaults to false, and controls whether we open a headless or
   /// a `headfull` browser.
-  Future<Chrome> launch(String url, { bool headless = false }) async {
+  ///
+  /// `skipCheck` does not attempt to make a devtools connection before returning.
+  Future<Chrome> launch(String url, { bool headless = false, bool skipCheck = false }) async {
     final String chromeExecutable = findChromeExecutable();
-    final Directory dataDir = fs.systemTempDirectory.createTempSync();
+    final Directory dataDir = fs.directory('.dart_tool');
```

Comment:
Should this be something more specific?

Reply:
Yeah, it fills up the folder with junk, will fix

Reply:
Fixed